live-ingester/src/main/java/org/alfresco/hxi_connector/live_ingester/domain/usecase/metadata/IngestMetadataCommand.java

```diff
@@ -31,12 +31,9 @@
 
 import java.util.Set;
 
-import org.springframework.validation.annotation.Validated;
-
 import org.alfresco.hxi_connector.live_ingester.domain.usecase.metadata.model.CustomPropertyDelta;
 import org.alfresco.hxi_connector.live_ingester.domain.usecase.metadata.model.PropertyDelta;
 
-@Validated
```

Comment:
Why fields are not validated any more?

Reply:
there is no validation with use of annotations so it is unnecessary - this annotation was left by my by mistake after testing jakarta validation annotations